lib/parser.js

```diff
@@ -225,15 +237,19 @@ var parser = {
                 } else if (node.parent && node.parent.type === 'VariableDeclarator') {
                   functionName = node.parent.id.name;
                 }
-                var fnoutLine = '\nconsole.log("debuguy,"' +
+                var fnoutLine = 'var DEBOUT="debuguy,"' +
```

Comment:
Please see the line @244 & @245
node.loc.start.line + '";console.log(DEBOUT + ",STR");';
var fnEndLine = 'console.log(DEBOUT + ",END");';